tools/acrn-manager/README.rst

```diff
@@ -93,6 +94,25 @@ Use the ``stop`` command to stop one or more running VM:
 
    # acrnctl stop vm-yocto vm1-14:59:30 vm-android
 
+RESCAN BLOCK DEVICE
+===================
+
+Use the ``blkrescan`` command to trigger a rescan of
+virtio-blk device by guest VM, inorder to revalidate and 
+update the backend file.
+
+.. code-block:: none
+
+   # acrnctl blkrescan vmname slot,newfilepath
+   vmname:     Name of VM with dummy backend file attached to virtio-blk device.
+   slot:       Slot number of the virtio-blk device.
+   newfilpath: File path for the backend of virtio-blk device.
+
+Note: blkrescan is only supported when VM is launched with 
```

Comment:
Can you use the `.. note::` directive here so it is properly formatted when we generate the HTML output?

Reply:
done